# Array of human clusters

std_msgs/Header header

# Array of clusters
HumanCluster[] clusters

# Total number of clusters
int32 num_clusters
